style.css

```diff
@@ -0,0 +1,204 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+
+html {
+  box-sizing: border-box;
+}
+
+*,
+*::before,
+*::after {
+  box-sizing: inherit;
+}
+
+body {
+  width: 20vw;
+  height: 83vh;
+}
+
+.navImg1 {
+  width: 16px;
+  border: 2px solid #6070ff;
+  height: 0;
+  position: absolute;
+  top: 63px;
+  left: 323px;
+  right: 36px;
+}
+
+.navImg2 {
+  width: 16px;
+  border: 2px solid #6070ff;
+  height: 0;
+  position: absolute;
+  top: 68px;
+  left: 323px;
+  right: 36px;
+}
+
+.navImg3 {
+  width: 16px;
+  border: 2px solid #6070ff;
+  height: 0;
+  position: absolute;
+  top: 73px;
+  left: 323px;
+  right: 36px;
+}
+
+.logo a {
+  position: absolute;
+  width: 76px;
+  height: 20px;
+  font-family: 'Poppins', Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 20px;
+  color: #6070ff;
+  top: 58px;
+  left: 24px;
+  right: 275px;
+  bottom: 640px;
+}
+
+.title-name {
+  position: absolute;
+  width: 327px;
+  height: 150px;
+  font-family: 'Poppins', Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  color: #172b4d;
+  top: 133px;
+  left: 24px;
+  right: 24px;
+  bottom: 362px;
+}
+
+.title-intro {
+  position: absolute;
+  width: 327px;
+  height: 146px;
+  font-family: 'Poppins', Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #344563;
+  top: 309px;
+  left: 24px;
+  right: 24px;
+  bottom: 204px;
+}
+
+.Backgroud {
+  position: absolute;
+  width: 392px;
+  height: 587.15px;
+  left: 0;
+  top: 104px;
+  background-image: url(image/background_img.png);
+  background-repeat: no-repeat;
+}
```

Comment:
To fix the change requested by the previous reviewer and to help you out here let's do something first let's remove these styles except `background-image`, `background-repeat`, and let's add replace with this once and it should fix the problem you have.

```
width: 100vw;
height: 100vh;
margin-top: 100px;
background-size: contain;

```